packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -557,7 +559,8 @@ export type SdkModelPropertyType =
   | SdkPathParameter
   | SdkBodyParameter
   | SdkHeaderParameter
-  | SdkCookieParameter;
+  | SdkCookieParameter
+  | SdkServiceResponseHeader;
```

Comment:
just curious what does `SdkModelPropertyType` stands for?

Reply:
all client types converted from typespec `ModelProperty` type will be `SdkModelPropertyType`.